utils/phpstan-extensions/tests/Rule/CheckNotTestsNamespaceOutsideTestsDirectoryRule/CheckNotTestsNamespaceOutsideTestsDirectoryRuleTest.php

```diff
@@ -21,7 +21,7 @@ public function testRule(string $filePath, array $expectedErrorsWithLines): void
 
     public function provideData(): Iterator
     {
-        yield [__DIR__ . '/Fixture/tests/TestsNamespaceOutsideTestsDirectoryClass.php', []];
+        yield [__DIR__ . '/Fixture/tests/TestsNamespaceInsideTestsDirectoryClass.php', []];
     }
 
```

Comment:
It is impossible to test outside of test directory because the test files are inside PHPStan Extension util test folder. Remember we had a call about this? I forgot to change the class name in my last PR after the discussion.

Reply:
I see :+1: